### ISSUE TITLE
Resume malfunctioning, vanishing episodes

### ISSUE THREAD

**Issue by cg00001** [kodifitzwell/repo#3]
Using Kodi Nexus with Arch Linux.
I tried resume on and off and Trakt options to Pov or Trakt but this issue remains.
I'm using Next Episodes, when I watch an episode and stop eg in the middle of it, if I exit Next Episodes and go back in, the episode (and the whole show) is vanished.
After that, if I go in my Pov - TV Collection, the show is there, but the episode is gone -see pic.
That means I have to use another addon to watch that episode, and watch future episodes to the end without exiting the player.
I'll install POV in a blank Kodi tomorrow to see if the issue persists and make a log.
![Pov](https://user-images.githubusercontent.com/38960370/235277325-a646e173-17b8-4be2-83fe-7c51328874a8.jpg)

**Comment by cg00001** [comment]
Kodi log:
[kodi.log](https://github.com/kodifitzwell/repo/files/11358321/kodi.log)

**Comment by kodifitzwell** [comment]
I can confirm the issue with Kodi 20 is not in 19.  I have a suspicion of what might have changed in 20 that is causing this.  I will update as soon as I can fix and test.

thanks for reporting!

**Comment by cg00001** [comment]
I did stop playback a couple of times by accident and it resumed fine without the issue above -I enabled Always resume.
Maybe the update on 30/4 fixed it?

**Comment by kodifitzwell** [comment]
yes, the 3.5.0 update was specific to fix the resume and missing episodes issues with Kodi 20 Nexus.

**Comment by cg00001** [comment]
Thank you, works fine.
